package.json

```diff
@@ -161,6 +161,7 @@
 		"sinon-chai": "^3.5.0",
 		"tslint": "^6.1.0",
 		"typescript": "^3.8.3",
+		"typescript-tslint-plugin": "^0.5.5",
```

Comment:
would be better to add in a different PR, it doesn't sound linked to "adding basic tests"

Reply:
I added the tslint plugin to help fix obvious problems in code and formatting. Keeping it here.